e2e/recipes/recipes_test.go

```diff
@@ -0,0 +1,422 @@
+// SPDX-FileCopyrightText: The RamenDR authors
+// SPDX-License-Identifier: Apache-2.0
+
+//nolint:testpackage
+package recipes_test
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	recipe "github.com/ramendr/recipe/api/v1alpha1"
+	"go.uber.org/zap"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/ramendr/ramen/e2e/app"
+	"github.com/ramendr/ramen/e2e/config"
+	"github.com/ramendr/ramen/e2e/deployers"
+	"github.com/ramendr/ramen/e2e/recipes"
+	"github.com/ramendr/ramen/e2e/test"
+	"github.com/ramendr/ramen/e2e/types"
+	"github.com/ramendr/ramen/e2e/workloads"
+)
+
+const (
+	ctxName         = "disapp-deploy-busybox-pvc"
```

Comment:
Why not use a shorter name that looks like the real name? e.g "disapp-deploy-rbd". This helps the reader to understand how this is related to real workload we test.